### PR TITLE
Updates README for usage with Azure Shell and adds fun Terraform

### PR DESCRIPTION
**Pull request by rwc**
* Bakes the packer functionality into bootstrap script in Terraform
* Removes the packer directory as it's no longer needed
* Updates the README with Azure Shell guide

Signed-off-by: Rob Campbell <rcampbell@chef.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+6. Copy the value for the key "tenantId," this will be your `azure_tenant_id`
+7. In the shell, clone the working repository by performing the following: `git clone https://github.com/stephenlauck/windows-habitat-no-docker.git`
+
+### Using Terraform
+We're going to use [Terraform](https://www.terraform.io/docs) to spin up the Windows 2016 workstation.
+
+1. After cloning the repo above, you'll want to do the following and save the changes:
+  ```shell
+  In the shell:
+  vim windows-habitat-no-docker/terraform/azure/terraform.tfvars
```
Comment (stephenlauck) [inline README.md]: you spelled emacs wrong.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
 
+# Getting Started in Azure Shell
+So you've decided to take the plunge into Habitat + Windows. You might be thinking, what on earth have I decided to do? Fret not, follow along and enjoy the path to habilightenment.
+
+## Requirements
+- Unencumbered internet connection
+- Browser
+- Time + Coffee
+- Microsoft Azure Account
+- Remote Desktop
```
Comment (stephenlauck) [inline README.md]: We should have a section on how to install RDP or at least a link to it?

File: README.md (hunk 1 above)
Comment (rwc) [reply README.md]: You're not wrong. I use `spacemacs` locally.

File: README.md (hunk 2 above)
Comment (rwc) [reply README.md]: Fixed in https://github.com/stephenlauck/windows-habitat-no-docker/pull/7

**Pull request merged**